launch/lidartag_vader.launch.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think we don't need this file - we can use at128.

Reply:
Addressed in [5d95553](https://github.com/tier4/LiDARTag/pull/15/commits/5d955532a2c6287e6ebc96ebb006bc36ec772650)